### PR TITLE
feat: Add team section on company show

### LINKED ISSUE #1106: On Company Show, I see a a Team section displaying a list of person linked to the company
https://github.com/twentyhq/twenty/assets/19412894/ccf34831-5795-439a-bbfd-0c4f471b5f68

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=7813%3A79050&mode=dev

This ticket scope is only to display the list of person attached to a company.
```[tasklist]
### Tasks
- [ ] Create a component in @/ui/company/components/CompanyTeam.tsx that is displaying a list of PeopleCard
- [ ] Create a component in @/ui/people/components/PeopleCard that is displaying the person with its displayName, Avatar and jobTitle
```

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #1106

Demo: 
https://www.loom.com/share/683ed55894254dec97bc04b60b5f9377?sid=64e569fd-4e7e-431b-a7e5-dd2bcadc3666

### CODE REVIEW

File: front/src/modules/companies/components/CompanyTeam.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledTitle = styled.div`
+  color: ${({ theme }) => theme.font.color.primary};
+  font-family: Inter;
+  font-size: 13px;
+
+  font-style: normal;
```
Comment (charlesBochet) [inline front/src/modules/companies/components/CompanyTeam.tsx:42]: should not be needed

File: front/src/modules/companies/components/CompanyTeam.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  font-weight: 500;
+  line-height: 150%;
```
Comment (charlesBochet) [inline front/src/modules/companies/components/CompanyTeam.tsx:44]: use theme :)

File: front/src/modules/people/components/PeopleCard.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import { Avatar } from '@/users/components/Avatar';
+import { Person } from '~/generated/graphql';
+
+export type PeopleCardPropsType = {
+  person: Pick<Person, 'id' | 'avatarUrl' | 'displayName' | 'jobTitle'>;
+};
+
+const StyledCard = styled.div`
+  align-items: center;
+  align-self: stretch;
```
Comment (charlesBochet) [inline front/src/modules/people/components/PeopleCard.tsx:13]: idem!

File: front/src/modules/people/components/PeopleCard.tsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  align-items: flex-start;
+  display: flex;
+  flex: 1 0 0;
+  flex-direction: column;
+`;
+
+const StyledTitle = styled.div`
+  color: ${({ theme }) => theme.font.color.primary};
+  font-family: Inter;
+  font-size: 13px;
```
Comment (charlesBochet) [inline front/src/modules/people/components/PeopleCard.tsx:36]: use theme or remove css properties (see CompanyTeam comments)

File: front/src/modules/people/components/PeopleCard.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+  font-style: normal;
+  font-weight: 500;
+  line-height: 150%;
+`;
+
+const StyledJobTitle = styled.div`
+  border-radius: ${({ theme }) => theme.spacing(1)};
+  color: ${({ theme }) => theme.font.color.secondary};
+  padding: 3px 4px 3px 0px;
```
Comment (charlesBochet) [inline front/src/modules/people/components/PeopleCard.tsx:46]: let's not use 3px for values actually. Let's use theme spacing too

File: front/src/modules/companies/components/CompanyTeam.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+`;
+
+const StyledTitleContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  backdrop-filter: blur(5px);
+  color: ${({ theme }) => theme.font.color.primary};
+  display: flex;
+  justify-content: space-between;
+  padding: 12px 12px 0px 12px;
```
Comment (charlesBochet) [inline front/src/modules/companies/components/CompanyTeam.tsx:25]: let's use theme spacing for padding

**Review by charlesBochet — COMMENTED** [review]
Nice work! Added cosmetic comments

**Comment by gitstart-twenty** [comment]
> 

On it

**Comment by gitstart-twenty** [comment]
Fixed companies with many members:


https://github.com/twentyhq/twenty/assets/140154534/b54ca5c3-60e0-4861-9ff3-6a95d8317c5c

File: front/src/modules/people/components/PeopleCard.tsx (hunk 3 above)
Comment (gitstart-twenty) [reply front/src/modules/people/components/PeopleCard.tsx:13]: We are actually using this one. Using `width` would result in something like below:

Screenshot 2023-08-09 at 23 45 35

**Review by charlesBochet — APPROVED** [review]
Thank you!

**Pull request merged**